### ISSUE TITLE
Directory Mapping not working 2.7.1

### ISSUE THREAD

**Issue by SoiledBrush** [equeim/tremotesf2#534]
I have single directory mount ```/data=/data```

For example: when I open file located in ```/data/storage/.Downloads/.arrs/sonarr``` I get error ```Unable to handle this kind of location``` in nautilus. 

When I try to make call manually
 ```
dbus-send --session --dest=org.freedesktop.FileManager1 --print-reply \
 --type=method_call /org/freedesktop/FileManager1 \
 org.freedesktop.FileManager1.ShowItems array:string:"file:///data/storage/.Downloads/.arrs/sonarr" string:""
```
Everything works fine. Tried both on aur package and flatpak versions of tremotesf.

**Comment by equeim** [comment]
Can you use dbus-monitor to see what arguments tremotesf passes to ShowItems call?

**Comment by equeim** [comment]
The command is `dbus-monitor --session --monitor interface='org.freedesktop.FileManager1'`

I suspect that the issue is that Tremotesf passes path without file:// prefix. Does dbus-send command fail if you remove it?

**Comment by SoiledBrush** [comment]
Yes, that's exactly it 
```string "/data/storage/.Downloads/.arrs```
The command in dbus-send doesn't fail, i guess it's the file manager that can't read that path.

**Comment by equeim** [comment]
Fixed in 2.7.2
